sdk/marketplace/Microsoft.Azure.Management.Marketplace/tests/Microsoft.Azure.Management.Marketplace.Tests.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Marketplace" Version="1.0.1" />
+    <PackageReference Include="Microsoft.Azure.Management.Marketplace" Version="1.0.2" />
```

Comment:
Your test project should depend on the local SDK project, not public package, so please remove this line.
```suggestion
```